ui/src/react-admin/modules/shared/components/CheckboxDropdownModal/CheckboxDropdownModal.tsx

```diff
@@ -255,6 +255,7 @@ export const CheckboxDropdownModal: FunctionComponent<CheckboxDropdownModalProps
 									)}
 									icon={'search' as IconName}
 									value={searchKeyword}
+									ariaLabel={tText('aria label')}
```

Comment:
This is a generic checkbox component, so it doesn't really make sense to have a single translation here for all checkbox modals in the application

Better to expose this label as a property on the component:
https://github.com/viaacode/react-admin-core-module/pull/276/files#diff-e07c239766f9a2b86dcf9aabdd42c2195820e24ff57488869d7e52e977875582L75
```suggestion
									ariaLabel={searchInputAriaLabel}
```